### ISSUE TITLE
Automatically add backslash to path if missing

### ISSUE THREAD

**Issue by thorstenmaly** [DNPLab/DNPLab#48]
Currently, to load topspin data the path has to end with a backslash:

```
path = '../data/topspin/'
data = dnplab.load(path, 'topspin',20)

```

It the backslash is missing the export won't work. Maybe we can check whether the path ends with a backslash and add it when it is missing.

**Comment by ylin00** [comment]
This issue will be address after approval of #44.

**Comment by thcasey3** [comment]
fixed in #44
